lib/js-preprocessor/index.js

```diff
@@ -8,7 +8,7 @@ var funnel = require('broccoli-funnel');
 module.exports = function(tree, options) {
   var generated = new GenerateComponents(tree, options);
 
-  var merged = mergeTrees([tree, generated]);
+  var merged = mergeTrees([tree, generated], { overwrite: true });
```

Comment:
This is the error I'm getting without it https://gist.github.com/HeroicEric/f8b25ee502bd3dfa59f5c5d8e18c2fa8

I'm not sure why yet but this function is being called twice.